src/corelibs/U2View/src/ov_msa/MSAEditor.h

```diff
@@ -180,19 +199,28 @@ protected slots:
     /** Shows 'Export Image' dialog. */
     void sl_exportImage();
 
+    void sl_multilineViewAction() override;
+
+    bool setMultilineMode(bool newmode) override {
+        multilineMode = newmode;
+        return getUI()->setMultilineMode(multilineMode);
+    }
+
 protected:
     QWidget* createWidget() override;
+
+    void initActions() override;
     bool eventFilter(QObject* o, QEvent* e) override;
     bool onObjectRemoved(GObject* obj) override;
     void onObjectRenamed(GObject* obj, const QString& oldName) override;
 
-    void addCopyPasteMenu(QMenu* m) override;
+    void addCopyPasteMenu(QMenu *m, uint uiIndex) override;
```

Comment:
Do not use unsigned types: use only signed types. Mix of signed and unsigned types in code leads to errors.

Reply:
Let's file new issue and I will replace a ton of these uses

Reply:
Nope, please fix it in the patch: 
UGENE (and QT too) never uses unsigned types for trivial use cases (like indexes).
The rename will take 5 minutes, please fix it before the merge